tst/EnergyPlus/api/TestDataTransfer.c

```diff
@@ -106,6 +106,11 @@ void afterZoneTimeStepHandler(EnergyPlusState state)
             wallConstruction == -1 || floorConstruction == -1) {
             exit(1);
         }
+
+        char *filePath = inputFilePath(state);
+        printf("Input file path accessed via API: %s\n", filePath);
+        free(filePath);
```

Comment:
Ok, but where's the getEMSGlobalVariableValue / setEMSGlobalVariableValue test at?

(and I suppose the python version would be nice too).